core/src/test/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -383,6 +383,11 @@ public static String randomRealisticUnicodeOfCodepointLength(int codePoints) {
         }
         return array;
     }
+    
+    public static String randomTimeValue() {
+        final String[] values = new String[]{"d", "H", "ms", "s", "S", "w"};
+        return randomIntBetween(0, 3) + randomFrom(values);
```

Comment:
yes I guess that is more of a fantasy related to the name of the method, although 0s or 0d could be corner cases.